datascience/tables.py

```diff
@@ -2052,6 +2052,29 @@ def annotate(axis, ticks):
 
         self._visualize(column_for_categories, labels, xticks, overlay, draw, annotate, width=width, height=height)
 
+
+    def group_bar(self, column_label, **vargs):
+        """Plot a bar chart for the table.
+
+        The values of the specified column are grouped and counted, and one
+        bar is produced for each group.
```

Comment:
Let's leave a note here that although this method is just a `group` followed by a `bar`, we use it for pedagogical purposes. That way it's a bit more clear why this method exists.